bundles/at.medevit.elexis.emediplan.ui/OSGI-INF/l10n/plugin.properties

```diff
@@ -0,0 +1,12 @@
+#Generated by ResourceBundle Editor (http://essiembre.github.io/eclipse-rbe/)
+#Properties file for at.medevit.elexis.emediplan.ui
+
+Bundle-Name = eMediplan Ui
+
+Bundle-Vendor = medevit.at
+
```

Comment:
Do not translate these values (Bundle-Name and vendor)

Reply:
Ok. i think it's safer too!